shacl12-core/index.html

```diff
@@ -4262,424 +4255,13 @@ <h2>Summary of SHACL Syntax Rules</h2>
 		<section id="shacl-shacl" class="appendix">
 			<h2>SHACL Shapes to Validate Shapes Graphs</h2>
 			<p>
-				The following shapes graph is intended to enforce many of the syntactic constraints related to SHACL Core in this specification. 
-				As such, it can be understood as a machine-readable version of a subset of those constraints,
-				and should be understood as normative.
-				If differences are found between the constraints expressed here and elsewhere in this specification,
-				that indicates an error in the following shapes graph.
-				Please see the <a href="https://www.w3.org/2017/shacl/errata">Errata Page</a> for an enumeration and analysis of possible errors that have been reported.
-        		This shapes graph is available at <a href="http://www.w3.org/ns/shacl-shacl">http://www.w3.org/ns/shacl-shacl</a>.
-        		That version may be more up-to-date than this specification as errata are noted against this specification.
+        		The shapes graph available at <a href="https://github.com/w3c/data-shapes/blob/gh-pages/shacl12-vocabularies/shacl-shacl.ttl">http://www.w3.org/ns/shacl-shacl</a>
+				is intended to enforce many of the syntactic constraints related to SHACL Core in this specification. 
+				As such, it can be understood as a machine-readable version of a subset of those constraints.
+			</p>
+			<p class="todo">
+				TODO: Replace the hyperlink to the official site once that has been updated.
```

Comment:
The new file can be included in this PR.

The TR is `/TR/shacl/index.html`. Associated files go in the same directory. The link is `href="./shacl-shacl.ttl"`.

Reply:
Ok thanks. Here my preference is to decouple the release cycles of these files, so that the shacl-shacl is not just about core but also about any future doc such as node expressions. This means it would be more like a living standard and live outside of Core.

Reply:
I would think that SHACL-SHACL's review scope should cover every class and predicate with the `sh:` prefix, so it makes sense to me that shacl-shacl would live outside of core.  Would it get a standalone document accompanying the TTL file?  If so, I'd guess it would be a pretty short document.